physionet-django/export/views.py

```diff
@@ -61,3 +64,49 @@ def get(self, request, project_slug, version, *args, **kwargs):
         project = get_object_or_404(PublishedProject, slug=project_slug, version=version)
         serializer = PublishedProjectDetailSerializer(project)
         return Response(serializer.data)
+
+
+class PublishedProjectSearch(mixins.ListModelMixin, generics.GenericAPIView):
+    """
+    Search for a Published Project using the get_content function inside Search Module's views.py
+    """
+
+    serializer_class = PublishedProjectSerializer
+
+    def get_queryset(self):
+        resource_type = self.request.GET.getlist('resource_type', 'all')
+        search_term = self.request.GET.get('search_term', ' ')
+
+        print(search_term, resource_type)
+
+        # Define a dictionary to map resource types to their corresponding integer values
```

Comment:
Could this be imported instead of hardcoded? It looks like we capture the types here:

https://github.com/MIT-LCP/physionet-build/blob/07c02703ca533ba379059d38dc4534fe99a1caa9/physionet-django/project/modelcomponents/coreproject.py#L57-L69

Reply:
Now getting these from `ProjectType`